README.md

```diff
@@ -1,2 +1,37 @@
 # Portfolio_and_mobile_first
-Portfolio with "Mobile First" approach 
+This a mobile version skeleton of portfolio.
+
+
+# Built With
+- HTML 
+- SCC
+
+# Description 
+- It should look almost like the Figma template I chose.
+- Linter should be set up correctly for this repository.
+
+
```

Comment:
- [Optional] Consider adding a "Getting started" section. This will make your work look more professional. This section should contain the steps required for one to get a local copy of your project running.